runtime/mandala/src/lib.rs

```diff
@@ -1211,18 +1210,25 @@ impl module_homa::Config for Runtime {
 }
 
 parameter_types! {
-	pub const KSM: CurrencyId = CurrencyId::Token(TokenSymbol::KSM);
-	pub const LKSM: CurrencyId = CurrencyId::Token(TokenSymbol::LKSM);
+	pub const DotCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::DOT);
+	pub const LdotCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::LDOT);
+	pub const MinimumMintThreshold: Balance = 1_000_000_000;
+	pub RelaychainSovereignSubAccount: MultiLocation = MultiLocation::X2( Junction::Parent, Junction::AccountId32 {
+		network: RelayNetwork::get(),
+		id: Utility::derivative_account_id(ParachainInfo::get().into_account(), RELAYCHAIN_SUB_ACCOUNT_ID).into(),
```

Comment:
Is this the right way?

Reply:
looks right. can you have a test to assert this value?

Reply:
We need to asset the derived account id is different from original one, and can be used with the origin one.

Reply:
Added an integration test to the Mandala runtime to assert the derivative_account_id() produces unique addresses